daphne/src/messages.rs

```diff
@@ -428,30 +555,40 @@ impl Decode for CollectReq {
 // TODO Add serialization tests.
 #[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
 pub struct CollectResp {
+    pub report_count: u64,
```

Comment:
Isn't this missing the query-specific response parameters, like the batch ID for chunky queries?

Reply:
Yes, this gets added later in the PR stack with a type called PartialBatchSelector. Should be clearer then.

Reply:
Is that PR coming? I only see one PR in this stack.

Reply:
Yes.

Reply:
I haven't created PRs yet because there a number of them and I have to  keep them rebased.